.github/workflows/test-windows.yml

```diff
@@ -0,0 +1,189 @@
+---
+name: Test Windows
+
+on:
+  pull_request:
+  workflow_dispatch:
+  merge_group:
+
+env:
+  QT_QPA_PLATFORM: minimal
+  # Last known good conan version
+  # 2.0.3 has a bug on Windows (conan-io/conan#13606)
+  CONAN_VERSION: 2.0.2
+
+concurrency:
+  group: test-windows-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-windows:
+    name: "Test ${{ matrix.os }}, Qt ${{ matrix.qt-version }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest]
+        qt-version: [5.15.2, 6.2.4]
+        force-lto: [false]
+        plugins: [false]
+        skip-artifact: [false]
+        skip-crashpad: [false]
+      fail-fast: false
+    env:
+      C2_BUILD_WITH_QT6: ${{ startsWith(matrix.qt-version, '6.') && 'ON' || 'OFF' }}
+      QT_MODULES: ${{ startsWith(matrix.qt-version, '6.') && 'qt5compat qtimageformats' || '' }}
+
+    steps:
+      - name: Force LTO
+        if: matrix.force-lto
+        run: |
+          echo "C2_ENABLE_LTO=ON" >> "$Env:GITHUB_ENV"
+        shell: pwsh
+
+      - name: Enable plugin support
+        if: matrix.plugins
+        run: |
+          echo "C2_PLUGINS=ON" >> "$Env:GITHUB_ENV"
+        shell: pwsh
+
+      - name: Set Crashpad
+        if: matrix.skip-crashpad == false
+        run: |
+          echo "C2_ENABLE_CRASHPAD=ON" >> "$Env:GITHUB_ENV"
+        shell: pwsh
+
+      - name: Set BUILD_WITH_QT6
+        if: startsWith(matrix.qt-version, '6.')
+        run: |
+          echo "C2_BUILD_WITH_QT6=ON" >> "$Env:GITHUB_ENV"
+        shell: pwsh
+
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          fetch-depth: 0 # allows for tags access
+
+      - name: Install Qt
+        uses: jurplel/install-qt-action@v3.3.0
+        with:
+          cache: true
+          cache-key-prefix: ${{ runner.os }}-QtCache-${{ matrix.qt-version }}-v2
+          modules: ${{ env.QT_MODULES }}
+          version: ${{ matrix.qt-version }}
+
+      - name: Enable Developer Command Prompt
+        if: startsWith(matrix.os, 'windows')
+        uses: ilammy/msvc-dev-cmd@v1.12.1
+
+      - name: Setup conan variables
+        if: startsWith(matrix.os, 'windows')
+        run: |
+          "C2_USE_OPENSSL3=$(if ($Env:C2_BUILD_WITH_QT6 -eq "on") { "True" } else { "False" })" >> "$Env:GITHUB_ENV"
+          "C2_CONAN_CACHE_SUFFIX=$(if ($Env:C2_BUILD_WITH_QT6 -eq "on") { "-QT6" } else { "`" })" >> "$Env:GITHUB_ENV"
+        shell: powershell
+
+      - name: Setup sccache
+        # sccache v0.5.3
+        uses: nerixyz/ccache-action@9a7e8d00116ede600ee7717350c6594b8af6aaa5
+        if: startsWith(matrix.os, 'windows')
+        with:
+          variant: sccache
+          # only save on the default (master) branch
+          save: ${{ github.event_name == 'push' }}
+          key: sccache-test-${{ matrix.os }}-${{ matrix.qt-version }}-${{ matrix.skip-crashpad }}
```

Comment:
I'm not 100% sure: Will these use the same cache as the ones in the `build.yml`.

Reply:
I don't know I more or less just copied it from the `build.yml`